### DISCUSSION TITLE
worldserver crash

### DISCUSSION THREAD

**Discussion by rsvr88** [azerothcore/azerothcore-wotlk#9143] [General]
Greetings,
worldserver crashes on Initializing Scripts... > Loading C++ scripts.
How can I fix it?

**Comment by rsvr88** [comment]
Made new build without module, now it's ok

**Comment by Watafackah** [comment]
Having the same error, Worldserver not working as expected, what module did you removed?

**Reply by rsvr88** [reply]
I don't remember all problem modules. I remember that I removed visual weapon, npc buffer, character tools, npc talent template.
Now I have transmog and eluna lua engine only.
